packages/semantic-dom-diff/get-diffable-html.js

```diff
@@ -129,7 +129,7 @@ export function getDiffableHTML(html, options = {}) {
   function getAttributeString(el, { name, value }) {
     if (shouldStripAttribute({ name, value })) return '';
     if (name === 'class') return ` class="${getClassListValueString(el)}"`;
-    return ` ${name}="${value}"`;
+    return ` ${name}="${value.replace(/&/g, '&amp;').replace(/"/g, '&quot;')}"`;
```

Comment:
Thanks for taking the time to review this Lars!

I've checked with the HTML spec for attribute values which specify:

> Attribute values are a mixture of text and character references, except with the additional restriction that the text cannot contain an ambiguous ampersand.

and

> Double-quoted attribute value syntax
>
>    The attribute name, followed by zero or more space characters, followed by a single U+003D EQUALS SIGN character, followed by zero or more space characters, followed by a single U+0022 QUOTATION MARK character ("), followed by the attribute value, which, in addition to the requirements given above for attribute values, must not contain any literal U+0022 QUOTATION MARK characters ("), and finally followed by a second single U+0022 QUOTATION MARK character (").

From: https://www.w3.org/TR/2011/WD-html5-20110525/syntax.html#attributes-0

So I think only escaping for these two characters is the way to go.

I have moved the RegExp and replacer function to a constant to avoid redefinition.